### ISSUE TITLE
System reliability can be more reliable than some critical subsystems depending on the archiitectural design

### ISSUE THREAD

**Issue by david-a-wheeler** [Derek-Jones/ESEUR-book#22]
In 6.5.1:

> A system is only as reliable as its least reliable critical subsystem

That's clearly not true, since error-correction mechanisms and redundancy can dramatically increase the reliability of a system even with less-reliable subsystems. Modify this, e.g.:

> In many systems, depending on their architectural designs, a system is only as reliable as its least reliable critical subsystem. ...

**Comment by Derek-Jones** [comment]
Yes, some nuance needs to be added to that overly broad assertion.

I will try to come up with a data driven solution.
